app/controllers/pieces_controller.rb

```diff
@@ -1,2 +1,24 @@
+require 'pry'
```

Comment:
Hmmmm. That's weird. I Don't even know what that is actually. ??

On Fri, Aug 14, 2015, 5:12 PM Stephen B. Thompson <notifications@github.com>
wrote:

> In app/controllers/pieces_controller.rb
> <https://github.com/Rails-Riders/chess/pull/17#discussion_r37119888>:
>
> > @@ -1,2 +1,24 @@
> > +require 'pry'
>
> This is just for debugging, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Rails-Riders/chess/pull/17/files#r37119888>.
>